module/Calculator/SpeedCalculator.h

```diff
@@ -52,9 +52,14 @@ class SpeedCalculator {
   double prevLeftMileage;
   int prevRightTime;
   int prevLeftTime;
+  // 回頭以外のPIDゲイン
   static constexpr double K_P = 0.004;
   static constexpr double K_I = 0.0000005;
   static constexpr double K_D = 0.0007;
+  // 回頭用PIDゲイン
```

Comment:
```suggestion
  // 回頭用PIDゲイン
  // TODO: 追々調整予定
```
調整予定であれば、その旨を記述するとより安全かと思いました。